### DISCUSSION TITLE
Use of UTF8 in Red including compiled

### DISCUSSION THREAD

**Discussion by AlanoDel** [red/red#5388] [Q&A] [answered]
Hi!
I have scripts in R2, but they not accept UTF8.
I need UTF8 for some languages, so my question is "Is it possible to show and analyse text in UTF8?"
Thanks in advance,
Alain

**Comment by hiiamboris** [comment, marked as answer]
By default text IO in Red (`read` and `write`) only supports UTF-8, so yes.
